src/services/api.js

```
@@ -13,22 +13,37 @@ export const fetchProducts = async () => {
-  //     }, 500);
-  //   });
-  // };
\ No newline at end of file
+export const login = async (loginId, password) => {
+  try {
+    const response = await api.post('/login', { loginId, password });
+    return response.data;
+  } catch (error) {
+    throw error;
+  }
+};
```

Biome

[error] The catch clause that only rethrows the original error is redundant.

These unnecessary catch clauses can be confusing. It is recommended to remove them.

(lint/complexity/noUselessCatch)